updateStableIds/src/main/java/org/reactome/release/update_stable_ids/UpdateStableIds.java

```diff
@@ -0,0 +1,163 @@
+		// 'excluded' classes were Regulation classes, the filter was not needed anymore.
+//		final List<String> acceptedClassesWithStableIdentifiers = Arrays.asList("Pathway","SimpleEntity","OtherEntity","DefinedSet","Complex","EntityWithAccessionedSequence","GenomeEncodedEntity","Reaction","BlackBoxEvent","CandidateSet","Polymer","Depolymerisation","Polymerisation","Drug","FailedReaction","EntitySet");
+//		List<String> excludedClassesWithStableIdentifiers = Arrays.asList("PositiveGeneExpressionRegulation","PositiveRegulation","NegativeRegulation","Requirement","NegativeGeneExpressionRegulation");
+//		ResultSet classesWithStableIdentifiers = dbaGkCentral.executeQuery("SELECT DISTINCT _class FROM DatabaseObject WHERE StableIdentifier IS NOT NULL", null);		
+
+		logger.info("Fetching all Event and PhysicalEntity instances");
+		// Get all Event and PhysicalEntity instances and combine them into one large List
+		Collection<GKInstance> eventInstances = dbaSlice.fetchInstancesByClass(Event);
+		Collection<GKInstance> physicalEntityInstances = dbaSlice.fetchInstancesByClass(PhysicalEntity);
+		List<GKInstance> sliceInstances = new ArrayList<GKInstance>();
+		sliceInstances.addAll(eventInstances);
+		sliceInstances.addAll(physicalEntityInstances);
+
+		int incrementCount = 0;
+		logger.info("Total instances to check: " + sliceInstances.size());
+		for (GKInstance sliceInstance : sliceInstances) {
+			logger.info("Checking " + sliceInstance);
+			GKInstance gkCentralInstance = (GKInstance) dbaGkCentral.fetchInstance(sliceInstance.getDBID());
+			GKInstance prevSliceInstance = (GKInstance) dbaPrevSlice.fetchInstance(sliceInstance.getDBID());
+			// Check if instance is new and that it exists on gkCentral (they could be deleted)
+			if (prevSliceInstance != null && gkCentralInstance != null) {
+				
+				// Compare number of 'Modified' instances between slices
+				Collection<GKInstance> sliceInstanceModified = sliceInstance.getAttributeValuesList(modified);
+				Collection<GKInstance> prevSliceInstanceModified = prevSliceInstance.getAttributeValuesList(modified);
+				if (sliceInstanceModified.size() > prevSliceInstanceModified.size()) {
+					// Make sure StableIdentifier instance exists
+					if (sliceInstance.getAttributeValue(stableIdentifier) != null && gkCentralInstance.getAttributeValue(stableIdentifier) != null) {
+						logger.info("\tIncrementing " + sliceInstance.getAttributeValue(stableIdentifier));
+						incrementStableIdentifier(sliceInstance, dbaSlice, sliceIE);
+						incrementStableIdentifier(gkCentralInstance, dbaGkCentral, gkCentralIE);
+						incrementCount++;
+					}
+				} else {
+					if (sliceInstanceModified.size() == prevSliceInstanceModified.size()) {
+//						System.out.println("No change between releases");
+					} else {
+						logger.warn("Unexpected: Number of 'Modified' instances in [" + sliceInstance.getDBID() + "] is fewer than in previous release");
+					}
```

Comment:
The if statement is empty, so it might be best to make line 73 an else if block like this:
```
else if (sliceInstanceModified.size() < prevSliceInstanceModified.size()) {
    logger.warn("...");
}
``